sr_multi_description/urdf/right_srhand_ur10_joint_limited.urdf.xacro

```diff
@@ -8,7 +8,12 @@
     <joint name="world_joint" type="fixed">
         <parent link="world"/>
         <child link="ra_base_link"/>
-        <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
+        <xacro:if value="$(arg initial_z)">
+        	<origin xyz="0.0 0.0 $(arg initial_z)" rpy="0.0 0.0 0.0"/>
+		</xacro:if>
+        <xacro:if value="$(arg initial_z)">
+        	<origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
+		</xacro:if>
```

Comment:
The indentation is not correct.

Also, it seems that this modification should also be applied to other urdf files (like the one for ur5)